instrumentation/logback/logback-appender-1.0/library/src/test/java/io/opentelemetry/instrumentation/logback/appender/v1_0/internal/LoggingEventMapperTest.java

```diff
@@ -57,7 +58,7 @@ void testSome() {
   void testAll() {
     // given
     LoggingEventMapper mapper =
-        new LoggingEventMapper(false, singletonList("*"), false, false, false);
+        new LoggingEventMapper(false, singletonList("*"), false, false, false, false);
```

Comment:
Are we missing test coverage for the new `true` case?

Reply:
It is tested in `OpenTelemetryAppenderTest`